checks/linux.py

```diff
@@ -0,0 +1,176 @@
+from .vars import *
+from .utils.utils import *
+
+
+def getSessionTypeLine(lines):
+    sessionType = search('Session Type:', lines)
+    if len(sessionType) > 0:
+        return sessionType[0]
+
+
+def getWindowSystemLine(lines):
+    windowSystem = search('Window System:', lines)
+    if len(windowSystem) > 0:
+        return windowSystem[0]
+
+
+def checkDistro(lines):
+    isDistroNix = search('Distribution:', lines)
+
+    if len(isDistroNix) <= 0:
+        return
+
+    distro = isDistroNix[0].split()
+    distro = distro[2:]
+    distro = ' '.join(distro)
+
+    return [LEVEL_INFO, distro, ""]
+
+
+def checkFlatpak(lines):
+    isFlatpak = search('Flatpak Runtime:', lines)
+
+    if len(isFlatpak) > 0:
+        return [LEVEL_INFO, "Flatpak",
+                "You are using the Flatpak. Plugins are available as Flatpak extensions, which you can find in your Distributions Software Center or via <code>flatpak search com.obsproject.Studio</code>. Installation of external plugins is not supported."]
```

Comment:
```suggestion
                "You are using the Flatpak. Plugins are available as Flatpak extensions, which you can find in your Distribution's Software Center or via <code>flatpak search com.obsproject.Studio</code>. Installation of external plugins is not supported."]
```

Or `distribution's`.

Reply:
Fixed